input/fsh/Profiles clinical datapoints/DisutilityOfCare.fsh

```diff
@@ -43,6 +43,7 @@ Description: "Represents the type and impact of a complication experienced by a
 * severity from ComplicationImpactVS (required)
 * code and subject and severity MS
 * extension contains DueToEx named dueTo 0..* MS
+* extension[dueTo].valueReference only Reference (Procedure)
```

Comment:
This will allow any Procedure to take its place and determining the previous procedure is not guaranteed to be accurate - what if it was really a pre-previous one that caused it?

In this case I'd just update the field description to help clarify the mapping - as an example see how I did `partOf` here:

![image](https://user-images.githubusercontent.com/110988/187127647-5d0ec493-9562-4e6c-955b-4af34be215cc.png)

https://github.com/HL7/fhir-ichom-breast-cancer-ig/blob/master/input/fsh/Profiles%20clinical%20datapoints/DisutilityOfCare.fsh#L9

Reply:
I see, the difficulty is that there can be all sorts of procedures that cause a complication. But the partOf is not necessarily the same as due to, right? Also, the complication is a condition that doesn't contain a partOf option.

Reply:
I agree, the partOf could be better replaced by https://hl7.org/fhir/extension-procedure-causedby.html for example.

I'm not saying you should use partOf in a Condition though - keep dueTo with a reference to the procedure, but update the description so it shows up in the IG (see the orange arrow)

Reply:
Got it!